db/schema.rb

```diff
@@ -90,10 +90,10 @@
     t.string   "processing_state",        default: "requested", null: false
     t.datetime "created_at",                                    null: false
     t.datetime "updated_at",                                    null: false
-    t.string   "reference_no"
-    t.boolean  "closed"
     t.boolean  "override_delivery_error", default: false
     t.string   "delivery_error_type"
+    t.string   "reference_no"
+    t.boolean  "closed"
```

Comment:
That seems like a… not-good position to be in. Is there any way to resolve those sorts of issues so they don't keep pestering you?